src/i965_drv_video.c

```diff
@@ -2742,6 +2743,15 @@ i965_CreateContext(VADriverContextP ctx,
 
     i965->current_context_id = contextID;
 
+    i965->bufmgr = drm_intel_bufmgr_gem_init(intel->fd, 4096);
+
```

Comment:
drm_intel_bufmgr_gem_init might fail, so a check for i965->bufmgr is needed.

Reply:
Ok, will take a closer look.